packages/expo-updates/scripts/create-manifest-android.gradle

```diff
@@ -17,6 +17,13 @@ def nodeExecutableAndArgs = config.nodeExecutableAndArgs ?: ["node"]
 def entryFile = config.entryFile ?: "index.js"
 def assetsFile = entryFile.take(entryFile.lastIndexOf('.')) + ".assets"
 
+def reactNativeDevServerPort() {
+    def value = project.getProperties().get("reactNativeDevServerPort")
+    return value != null ? value : "8081"
```

Comment:
In this case I think value ?: “8081” does what we want. Other falsy values aren’t valid ports and port 0 typically means “pick a port for me” which is what we’re doing here.